src/component/App/App.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { HashRouter as Router, Route } from 'react-router-dom';
+
+import CountryInformation from '../CountryInformation/CountryInformation';
+import СountriesList from '../CountriesList/CountriesList';
+import Footer from '../Footer/Footer';
+import GlobalStyle from './GlobalStyle';
+import RegistrationForm from '../RegistrationForm/RegistrationForm';
+import Header from '../Header/Header';
+import Loader from '../Loader/Loader.jsx';
+import ScrollToTop from '../ScrollToTop/ScrollToTop';
+
+interface AppProps {
+  data: Array<Object>,
+  countries: Array<Object>,
+  language: string
+}
+
+const App: React.FC<AppProps> = ({ data, countries, language }) => {
+
+
+
+  if (data.length === 0 || language === null) {
```

Comment:
better having `isFetching` global flag for that purposes or declarative func like:

```
const isDataFetching = data => data .length ===0 || ...`

if (isDataFetching(data)) {
   
}
```